src/ssl_certman.c

```diff
@@ -704,6 +697,13 @@ int CM_VerifyBuffer_ex(WOLFSSL_CERT_MANAGER* cm, const unsigned char* buff,
     }
 #endif
 
+#ifdef HAVE_CRL
```

Comment:
I'm certain this will break users who want to override a CRL failure. See if you can find a better way to resolve customer issue overriding error with a custom OID.